+nosnoc/+time_freezing/cls_inelastic.m

```diff
@@ -0,0 +1,189 @@
+function [pss_model] = cls_inelastic(cls_model, opts)
+    import casadi.*
+    dims = cls_model.dims;
+    pss_model = nosnoc.model.Pss();
+    pss_model.dims = dims;
+
+    % transfer all common properties to pss_model
+    common_props = properties('nosnoc.model.Base');
+    for ii=1:length(common_props)
+        prop = common_props{ii};
+        pss_model.(prop) = cls_model.(prop);
+    end
+    
+    dims.n_contacts = length(cls_model.f_c);
+    % check dimensions of contacts
+    if isempty(dims.n_dim_contact)
+        warning('nosnoc: Please n_dim_contact, dimension of tangent space at contact (1, 2 or 3)')
```

Comment:
can it be 3?

Reply:
No. This is copy and pasted from the old tf code so :shrug:. removed.